src/main/java/Task.java

```diff
@@ -0,0 +1,26 @@
+public class Task {
+    protected String description;
```

Comment:
Do we have to use protected here? Would using private suffice?

Reply:
Yes I forgot to change this to private. Thanks for pointing that out!